### PR TITLE
added saw_variation_for_experiment method

### PR DESCRIPTION
**Pull request by srklonaris**
added methods for determining if a user has seen a variation for an experiment and for determining the time the saw it

### CODE REVIEW

File: lib/vanity/adapters/redis_adapter.rb [hunk 1]
```diff
@@ -209,6 +213,12 @@ def destroy_experiment(experiment)
         @experiments.del *alternatives unless alternatives.empty?
       end
 
+      def get_saw_variation_time(experiment, identity)
```
Comment (phillbaker) [inline lib/vanity/adapters/redis_adapter.rb]: I'm not sure if there's a strict policy about maintaining consistency between the adapters, but it'd probably be good to add support for this to the other ones as well.

**Comment by phillbaker** [comment]
Thanks for the PR! This looks useful - can you explain a bit more of you specific usecase? 

It looks like this is failing on Travis.

**Comment by srklonaris** [comment]
Thanks for checking it out.  I will def add consistency between adapters.
The use case is to add a method to determine if a user has been assigned a
variation.  In my code that I want to ab test I want to do some other stuff
but only if a user has been assigned a variation.

What exactly is Travis?

**Comment by srklonaris** [comment]
never mind, i see, Ill look into it

**Comment by phillbaker** [comment]
@srklonaris just curious, does the [Playground#participant_info](https://github.com/assaf/vanity/blob/master/lib/vanity/playground.rb#L95) method address this? It returns an experiments and assignments for a given participant.

**Pull request closed without merging**